### PR TITLE
Adding API ideas from lunch

### PR DESCRIPTION
**Pull request by kris-nova**
@Grillz please review and merge

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,54 @@
...
+
 Amplify your developer impact by developing directly in Kubernetes.
+
+## kamp run
+
+
+#### Run an arbitrary docker container in Kubernetes (default to /bin/bash)
+
+```bash
+kamp run <image>:<tag>
```
Comment (Grillz) [inline README.md]: should we add `kamp run -f /path/to/Dockerfile` ?

Comment (Grillz) [inline README.md]: or better `kamp run <image_name>:<tag> -f /path/to/Dockerfile`, where you can optionally tag the image being built

Comment (kris-nova) [inline README.md]: Good call

File: README.md [hunk 2]
```diff
@@ -1,2 +1,54 @@
...
+```
+
+## kamp push
+
+#### Tag the container that is running in Kubernetes, and push it to a docker registry
+
+```bash
+kamp push <newimage>:<newtag>
+```
+
```
Comment (Grillz) [inline README.md]: may want a `kamp status` as well, just returning a `describe` for the pod

Comment (kris-nova) [inline README.md]: done

**Comment by Grillz** [comment]
We should consider allowing all of these commands to look first for a Dockerfile in `./` and assume they are wanting to run the commands for that container. This may require a small bit of local storage

**Comment by kris-nova** [comment]
Agree - I can code it as such and update the readme once the cobra commands are stubbed out - sound good?

**Comment by Grillz** [comment]
LGTM

**Pull request merged**